app/controllers/conferences_controller.rb

```diff
@@ -37,8 +34,8 @@ def redirect
 
   private
 
-  def generate_gid(conference)
-    conference.gid = Time.now.getutc
+  def generate_gid(team)
+    "2017-#{Time.now.getutc.to_i}-#{team.id}"
```

Comment:
Yes, much better to put the team in instead of the user id. However, the year should be hardcoded but retrieved dynamically (`Season.current.name`).